db/repos/accounts.js

```diff
@@ -255,7 +255,8 @@ class AccountsRepository {
 	 * @todo Add description for the return value
 	 */
 	insert(data) {
-		return this.db.none(this.pgp.helpers.insert(data, this.cs.insert));
+		const query = () => this.pgp.helpers.insert(data, this.cs.insert);
+		return this.db.none(query);
 	}
```

Comment:
@vitaly-t You made same change into another PR. 

https://github.com/LiskHQ/lisk/pull/1720/files#diff-bb3f5b06bf2cf765f0688199caad2593R258

Also both of the PR are related tissue state that its a change for transactions, and you are changing the accounts here. I believe this PR contains duplicate code from #1720. Please clean the PR so we can review the changes respective to the issue it is addressing.

Reply:
See my notes above, - this is a cumulative update ;) Once the source PR is merged, the other changes will disappear automatically ;)